app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __repr__(self):
+        return "{" + f"Name: {self.name}," \
+                     f" Health: {self.health}," \
+                     f" Hidden: {self.hidden}" + "}"
+
+    def __init__(self, name, health=100, hidden=False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+
+class Herbivore(Animal):
+
+    def hide(self):
+        if self.hidden:
+            self.hidden = False
+        else:
+            self.hidden = True
+
+
+class Carnivore(Animal):
+
+    @staticmethod
+    def bite(other):
```

Comment:
What does it mean other? Who is other? Other like alien?
I think you got point, other bad name in this case, because we know other is Animal who catched by Carnivore
So we can write like
`victim: Herbivore(or Animal, i don't sure what's better)`